torch/_dynamo/output_graph.py

```diff
@@ -552,6 +552,20 @@ def real_value_cache(self):
     def bound_symbols(self):
         return self.current_tracer.bound_symbols
 
+    # Unbacked symbols may be created as a function call in parent (e.g. t.to_list()).
+    # We track the symbol-proxy mapping for unbacked symbols when the
+    # SymNodeVariable is constructed (e.g. t.to_list() call) so that when subtracer tries
+    # to bind an unbacked symint, e.g. a subgraph input with shape u0,
+    # the proxy for u0 can be found in parent's bound_symbols.
+    def track_unbacked_symbols(self, s: torch.SymInt, proxy: torch.fx.Proxy):
```

Comment:
Can you assert that the proxy's fx.Tracer is the current SubgraphTracer's fx.Tracer?

Reply:
nvm, I see what's going on